content/250-postgres/350-integrations/100-netlify.mdx

```diff
@@ -216,8 +222,7 @@ As mentioned [above](#generate-prisma-client-in-a-postinstall-script-in-packagej
     },
     "dependencies": {
         "@netlify/blobs": "^8.1.0",
-        "@prisma/client": "^6.3.0",
-        "@prisma/extension-accelerate": "^1.2.1",
```

Comment:
If the Netlify integration is not being updated to set the direct connection instead of the Accelerate one, this may not be correct.

Would you please check with Product?